tests/api.c

```diff
@@ -25651,9 +25651,14 @@ static int test_wc_ecc_rs_to_sig(void)
             ret = WOLFSSL_FATAL_ERROR;
         }
     }
-
     if (ret == 0) {
-        ret = wc_ecc_sig_to_rs(sig, siglen, r, &rlen, s, &slen);
+       ret = wc_ecc_sig_to_rs(sig, siglen, r, &rlen, s, &slen);
```

Comment:
This test case is meant to use a good signature.
It looks like the FIPS code is modifying sig in a bad case.
Move this up to after the original wc_ecc_rs_to_sig().

Reply:
That's a great resolution to the problem!  the change will be in shortly.  Thanks!